nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -5,6 +5,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+    <PropertyGroup>
+        <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
```

Comment:
Do you support MSBuild versions below 16.0? If not, delete this entirely: we do it automatically for you as of microsoft/msbuild#3605. If you do, please condition these out: unnecessarily adding to this property has been observed to cause a fair amount of wasted memory in VS scenarios.

Reply:
We do support VS2017 as far as I know which is why I added this.

Reply:
Like this?